.travis.yml

```diff
@@ -9,9 +9,9 @@ matrix:
     - env: ENABLE_STYLE_TESTING=ON WITH_GUI=ON
       compiler: gcc
 ## split class vs. TOPP tests for GCC (otherwise it will run into time limit of 50min)
-    - env: ENABLE_STYLE_TESTING=OFF ENABLE_TOPP_TESTING=ON ENABLE_CLASS_TESTING=OFF WITH_GUI=ON
+    - env: ENABLE_STYLE_TESTING=OFF ENABLE_TOPP_TESTING=ON ENABLE_CLASS_TESTING=OFF WITH_GUI=ON DISABLE_PPWAVELET3=ON
```

Comment:
adding the ENV var is not enough, you need to modify the travis sh file and pass an option as well.